### ISSUE TITLE
IMMEDIATE ACTION REQUIRED: GEMA

### ISSUE THREAD

**Issue by mariapaulafn** [decentralizedcultureroom/musicroom#6]
Hi @victoriapriester we just found this in the Factory contract. Without this done now (we are two weeks away the event) we cant have performances. Please @chrishobcroft let victoria know by phone if she doesnt see it today, we cannot do anything about the contract. 
<img width="836" alt="bildschirmfoto 2018-08-24 um 18 41 58" src="https://user-images.githubusercontent.com/27913589/44597715-deee5e80-a7d0-11e8-80c4-42eafe62cc72.png">

**Comment by mariapaulafn** [comment]
Filed will let you know if it pulls through

**Comment by victoriapriester** [comment]
@mariapaulafn 
So as I undrestand, I have to include payment to Gema to the fees from my budget for the music room? 
Another question, how are the artists gonna be paid? Cash or transfer? It would be great if I could pay them at the place after their performance.
Let me know when the gema thing comes trough and maybe u can send me the document if there's any. I will take care from now on of the forms for the artists and songs.
Another one question - is the Factory gonna be open for people form outside? so anyone can come in to visit the venue?

**Comment by victoriapriester** [comment]
@mariapaulafn 
Is it possible to cover the invoice of some artists, so don't have a freelance in germany, by the non-profit company from czech? They would pay them on a contract base. 
Or is it possible someone else make an invoice and handle the payment for them?

**Comment by mariapaulafn** [comment]
@victoriapriester no, original artists are protected by GEMA. As long as there is no sampling or DJ prods, we are safe with the GEMA.

**Comment by mariapaulafn** [comment]
Factory: no, this is a closed event. We need the list of artists. Nobody can bring a +1.

**Comment by mariapaulafn** [comment]
Invoices: see this https://github.com/decentralizedcultureroom/concept/issues/4
All is paid by bank transfer. The non-profit can issue an invoice, nobody can invoice for the artist, it has to come from them.

**Comment by victoriapriester** [comment]
I don't understand. U mean we don't have to pay gema?
Okay, so no possibility of some "manager" invoicing for the artist?

**Comment by mariapaulafn** [comment]
No, we don't have to pay GEMA unless we use songs that dont belong to us. We just have to let them know. This is due to the nature of the non-profit, educational and private event. 
No, but the Czech non-profit can invoice.

**Comment by victoriapriester** [comment]
Ah, so the non profit company could cover some invoices?

**Comment by mariapaulafn** [comment]
Non-profits CAN invoice of course! :)

**Comment by victoriapriester** [comment]
Perfect!
